src/middlewared/middlewared/plugins/disk.py

```diff
@@ -1023,7 +1023,17 @@ async def sync_all(self, job):
         ):
             return
 
+        for i in range(10):
+            if i > 0:
+                await asyncio.sleep(1)
+
+            if await self.middleware.call('device.devd_connected'):
+                break
+        else:
+            self.logger.warning('Starting disk.sync_all when devd is not connected yet')
+
         sys_disks = list((await self.middleware.call('device.get_info', 'DISK')).keys())
+        self.logger.info('Found disks: %r', sys_disks)
```

Comment:
Hmm how about job.logs_fd.write so we dont "pollute" logs? Too wierd?

Reply:
Yes, it would be hard to find it there and match with other events. It'll only be one line on system boot / pool create/destroy / disk unplug, I don't see an issue here.

I've imagined `job.logs_fd` to be like separate log file for big external program output (e.g. `rclone`)